tests/script/test_add_preload_script.py

```diff
@@ -546,58 +533,65 @@ async def test_preloadScript_add_sandbox(websocket, context_id, html):
 
 
 @pytest.mark.asyncio
-@pytest.mark.parametrize("with_reload", [True, False],
-                         ids=["with reload", "without reload"])
-async def test_preloadScript_add_runImmediately(websocket, html, context_id,
-                                                with_reload):
-
-    await goto_url(websocket, context_id, html())
-
+async def test_preloadScript_add_sandbox_existing_context(
+        websocket, context_id, html):
+    # Add preload script, which checks if the page script was already executed.
     await execute_command(
         websocket, {
             "method": "script.addPreloadScript",
             "params": {
-                "functionDeclaration": "() => { window.foo='bar'; }",
+                "functionDeclaration": "() => { "
+                                       "window.SOME_VAR=["
+                                       "...(window.SOME_VAR ?? []), "
+                                       "'MY_SANDBOX_PRELOAD_SCRIPT']; }",
```

Comment:
It could be more readable to use `"""` here.

(If you agree, please change all occurrences)

Reply:
No objections. Done.